content/docs/install/manual-install/index.md

```diff
@@ -25,7 +25,7 @@ and [Automated Operations](../../concepts/automated_operations/) use cases. To i
     ```
     helm repo add keptn https://charts.keptn.sh
     helm repo update
-    helm install keptn keptn/keptn -n keptn --version=$KeptnVersion --create-namespace --set=continuous-delivery.enabled=true
+    helm install keptn keptn/keptn -n keptn --version=$KeptnVersion --create-namespace --set=continuousDelivery.enabled=true
```

Comment:
Actually, I think this file will be deleted by https://github.com/keptn/keptn.github.io/pull/1287 .  Is that correct?  I guess we can look at everything after we merge everything and adjust accordingly